projects/rocprofiler-sdk/source/lib/python/rocpd/csv.py

```diff
@@ -23,14 +23,392 @@
+
+    # Define mapping of output column name to JSON key
+    json_keys = [
+        "node_id",
+        "logical_node_id",
+        "cpu_cores_count",
+        "simd_count",
+        "cpu_core_id_base",
+        "simd_id_base",
+        "max_waves_per_simd",
+        "lds_size_in_kb",
+        "gds_size_in_kb",
+        "num_gws",
+        "wave_front_size",
+        "num_xcc",
+        "cu_count",
+        "array_count",
+        "num_shader_banks",
+        "simd_arrays_per_engine",
+        "cu_per_simd_array",
+        "simd_per_cu",
+        "max_slots_scratch_cu",
+        "gfx_target_version",
+        "vendor_id",
+        "device_id",
+        "location_id",
+        "domain",
+        "drm_render_minor",
+        "num_sdma_engines",
+        "num_sdma_xgmi_engines",
+        "num_sdma_queues_per_engine",
+        "num_cp_queues",
+        "max_engine_clk_ccompute",
+        "max_engine_clk_fcompute",
+        "sdma_fw_version.uCodeSDMA AS Sdma_Fw_Version",
+        "fw_version.uCode AS Fw_Version",
+        "cu_per_engine",
+        "max_waves_per_cu",
+        "workgroup_max_size",
+        "family_id",
+        "grid_max_size",
+        "local_mem_size",
+        "hive_id",
+        "gpu_id",
+        "workgroup_max_dim.x AS Workgroup_Max_Dim_X",
+        "workgroup_max_dim.y AS Workgroup_Max_Dim_Y",
+        "workgroup_max_dim.z AS Workgroup_Max_Dim_Z",
+        "grid_max_dim.x AS Grid_Max_Dim_X",
+        "grid_max_dim.y AS Grid_Max_Dim_Y",
+        "grid_max_dim.z AS Grid_Max_Dim_Z",
+        "vendor_name",
+        "product_name",
+    ]
+
+    # Build SELECT clause for json_extract columns
+    select_json = []
+    for column in json_keys:
+        match = re.match(r"(.+?)\s+AS\s+(.+)", column, re.IGNORECASE)
+        column_name, column_alias = (
+            (match.group(1), match.group(2)) if match else (column, column)
+        )
+        select_json.append(f"json_extract(extdata, '$.{column_name}') AS {column_alias}")
+
+    capabilities = [
+        "HotPluggable",
+        "HSAMMUPresent",
+        "SharedWithGraphics",
+        "QueueSizePowerOfTwo",
+        "QueueSize32bit",
+        "QueueIdleEvent",
+        "VALimit",
+        "WatchPointsSupported",
+        "WatchPointsTotalBits",
+        "DoorbellType",
+        "AQLQueueDoubleMap",
+        "DebugTrapSupported",
+        "WaveLaunchTrapOverrideSupported",
+        "WaveLaunchModeSupported",
+        "PreciseMemoryOperationsSupported",
+        "DEPRECATED_SRAM_EDCSupport",
+        "Mem_EDCSupport",
+        "RASEventNotify",
+        "ASICRevision",
+        "SRAM_EDCSupport",
+        "SVMAPISupported",
+        "CoherentHostAccess",
+        "DebugSupportedFirmware",
+        "PreciseALUOperationsSupported",
+        "PerQueueResetSupported",
+    ]
+
+    # Build SELECT clause for Capability columns
+    select_capability = []
+    for capability in capabilities:
+        select_capability.append(f"json_extract(extdata, '$.capability.{capability}')")
```

Comment:
```suggestion
        select_capability.append(f"json_extract(extdata, '$.capability.{capability}') AS 'Cap_{capability}'")
```
The capability titles are showing the "json_extract()" as the title.  We should alias them as "Cap_<name>"

Reply:
Fixed.